src/main/java/org/junit/runners/BlockJUnit4ClassRunner.java

```diff
@@ -335,6 +335,7 @@ protected Object runReflectiveCall() throws Throwable {
 
         @Nullable Statement statement = methodInvoker(method, test);
         /*
+          This is a false positive.
```

Comment:
By looking at the comment, it is hard for me to tell what error did the IntelliJ issue. Is methodInvoker return type incompatible with `@Nullable` statement type?